firestore/src/main/java/com/example/firestore/snippets/RetrieveDataSnippets.java

```diff
@@ -45,16 +45,51 @@ void prepareExamples() throws Exception {
     // [START fs_retrieve_create_examples]
     CollectionReference cities = db.collection("cities");
     List<ApiFuture<WriteResult>> futures = new ArrayList<>();
-    futures.add(cities.document("SF").set(new City("San Francisco", "CA", "USA", false, 860000L,
-        Arrays.asList("west_coast", "norcal"))));
-    futures.add(cities.document("LA").set(new City("Los Angeles", "CA", "USA", false, 3900000L,
-        Arrays.asList("west_coast", "socal"))));
-    futures.add(cities.document("DC").set(new City("Washington D.C.", null, "USA", true, 680000L,
-        Arrays.asList("east_coast"))));
-    futures.add(cities.document("TOK").set(new City("Tokyo", null, "Japan", true, 9000000L,
-        Arrays.asList("kanto", "honshu"))));
-    futures.add(cities.document("BJ").set(new City("Beijing", null, "China", true, 21500000L,
-        Arrays.asList("jingjinji", "hebei"))));
+    futures.add(
+            cities.document("SF")
+                    .set(new City(
+                            "San Francisco",
+                            "CA",
+                            "USA",
+                            false,
+                            860000L,
+                            Arrays.asList("west_coast", "norcal"))));
+    futures.add(
+            cities.document("LA")
+                    .set(new City(
+                                "Los Angeles",
+                                "CA",
+                                "USA",
+                                false,
+                                3900000L,
+                                Arrays.asList("west_coast", "socal"))));
+    futures.add(
+            cities.document("DC")
+                    .set(new City(
+                            "Washington D.C.",
+                            null,
+                            "USA",
+                            true,
+                            680000L,
+                            Arrays.asList("east_coast"))));
+    futures.add(
+            cities.document("TOK")
+                    .set(new City(
+                            "Tokyo",
+                            null,
+                            "Japan",
+                            true,
+                            9000000L,
+                            Arrays.asList("kanto", "honshu"))));
+    futures.add(
+            cities.document("BJ")
+                    .set(new City(
+                            "Beijing",
+                            null,
+                            "China",
+                            true,
+                            21500000L,
+                            Arrays.asList("jingjinji", "hebei"))));
```

Comment:
Applying this formatting would cause this code snipped to be longer than a single screen when viewed in the documentation. We want to keep the whole context visible on a single screen.